docs/responders/ropensci/invite_author.md

```diff
@@ -0,0 +1,32 @@
+ROpenSci :: Invite author
+=========================
+
+This responder is used by the author of an approved package to receive an invitation to join the team managing the new transfered package. Usually this invitation is sent automatically when the package is approved but it expires in a week. This responder allows the author to have the invitation sent again.
+
+## Listens to
+
+```
+@botname invite me to ropensci/package-name
+```
+
+## Requirements
+
+The _ropensci/package-name_ team must exist and the _package-name_ must be a package already transfered to rOpenSci, otherwise an error message will be sent as reply.
```

Comment:
At the point the author receives the new invitation to the team and org, they cannot have already transferred the package as they need to be organization member for that?

Reply:
yes indeed!
thank you!

Reply:
Done!